client/assets/stylesheets/jetpack-cloud.scss

```diff
@@ -0,0 +1,33 @@
+// Color Schemes
+// Currently used only by Calypso dev badge shown in development mode
+@import '~@automattic/calypso-color-schemes/src/calypso-color-schemes';
+
+// Shared
+@import 'shared/reset'; // css reset before the rest of the styles are defined
+@import './shared/loading';
+@import './gutenberg-base-styles';
+@import './shared/mixins/placeholder'; // Contains the placeholder mixin
+@import './shared/animation'; // Needed for the placeholder
+
+// Gutenberg resets
+// Not an actual "CSS reset", but an opiniated base style for HTML elements in Gutenberg views.
+:root {
+	@include reset;
+}
+
+// Change loading logo's color while Calypso boots
+.wpcom-site__logo {
+	fill: $light-gray-500;
+}
+
+// Hide Calypso header while Calypso boots
+.masterbar {
+	display: none;
+}
+
+// Gutenberg default typography
+html,
+body {
+	font-family: $default-font;
+	font-size: $default-font-size;
+}
```

Comment:
We should add our own additional reset and base styles here. Ideally, they should be split into sections and components.